src/main/java/org/apache/commons/math4/stat/descriptive/WeightedEvaluation.java

```diff
@@ -36,7 +41,7 @@
      * do not match, weights contain NaN, negative or infinite values, or
```

Comment:
Should `weights` not therefore have an `@NonNegative` annotation on its component type?